packages/sdk/src/Campaigns.ts

```diff
@@ -18,43 +20,54 @@ export class Campaigns {
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#create-campaign
 	 */
-	public create(campaign: $FixMe) {
-		return this.client.post('/campaigns', campaign)
+	public create(campaign: T.CampaignsCreateCampaign) {
+		return this.client.post<T.CampaignsCreateCampaignResponse>('/campaigns', campaign)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#update-campaign
 	 */
-	public update(nameOrId: $FixMe, campaign: $FixMe) {
-		return this.client.put(`/campaigns/${encode(nameOrId)}`, campaign)
+	public update(nameOrId: string, campaign: T.CampaignsUpdateCampaign) {
+		return this.client.put<T.CampaignsUpdateCampaignResponse>(`/campaigns/${encode(nameOrId)}`, campaign)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#get-campaign
 	 */
-	public get(name: $FixMe) {
-		return this.client.get(`/campaigns/${encode(name)}`)
+	public get(name: string) {
+		return this.client.get<T.CampaignsGetCampaignResponse>(`/campaigns/${encode(name)}`)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#delete-campaign
 	 */
-	public delete(name: $FixMe, params?: { force: boolean }) {
-		return this.client.delete(`/campaigns/${encode(name)}`, { force: !!params?.force })
+	public delete(name: string, params?: T.CampaignsDeleteParams) {
+		return this.client.delete(`/campaigns/${encode(name)}`, params)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#add-voucher-to-campaign
 	 */
-	public addVoucher(campaignName: string, body: $FixMe = {}) {
-		return this.client.post(`/campaigns/${encode(campaignName)}/vouchers`, body)
+
+	public addVoucher(name: string, params: T.CampaignsAddVoucherParams = {}) {
```

Comment:
Well, according to the docs:
https://docs.voucherify.io/reference#add-voucher-to-campaign
You can have `query_params` containing field: `vouchers_count`. Di you try this out? I don't see a way to pass this to the method. Additionally please notice that when you pass `vouchers_count` >= 2 then response is completely different.
I am not sure that the typings for `CampaignsAddVoucherParams` are correct - it's missing `code_config` - take a look at: https://docs.voucherify.io/reference#add-voucher-to-campaign

Reply:
Also this should not be called `params` but `body` since it's used as body of POST request.